Mage.Client/src/main/java/org/mage/plugins/card/dl/sources/ScryfallImageSupportTokens.java

```diff
@@ -350,6 +350,18 @@
             put("THB/Wolf", "https://api.scryfall.com/cards/tthb/11/en?format=image");
             put("THB/Zombie", "https://api.scryfall.com/cards/tthb/7/en?format=image");
 
+            // IKO
+            put("IKO/Emblem Narset Of The Ancient Way", "https://api.scryfall.com/cards/tiko/12/en?format=image");
+            put("IKO/Beast", "https://api.scryfall.com/cards/tiko/10/en?format=image");
+            put("IKO/Cat Bird", "https://api.scryfall.com/cards/tiko/2/en?format=image");
+            put("IKO/Cat", "https://api.scryfall.com/cards/tiko/1/en?format=image");
+            put("IKO/Dinosaur Beast", "https://api.scryfall.com/cards/tiko/11/en?format=image");
+            put("IKO/Dinosaur", "https://api.scryfall.com/cards/tiko/8/en?format=image");
+            put("IKO/Feather", "https://api.scryfall.com/cards/tiko/9/en?format=image");
+            put("IKO/Human Soldier", "https://api.scryfall.com/cards/tiko/4/en?format=image");
```

Comment:
There are 3 different human soldiers, not one. Must be:
* `IKO/Human Soldier/1`
* `IKO/Human Soldier/2`
* `IKO/Human Soldier/3`